ng2-components/ng2-activiti-form/src/components/widgets/amount/amount.widget.scss

```diff
@@ -5,11 +5,23 @@
     &-amount-widget {
         width: 100%;
         vertical-align: baseline !important;
-        padding-left: 14px;
+
+        .mat-input-element {
+            margin-left: 11px;
+        }
     }
 
     &-amount-widget__container {
         max-width: 100%;
+
+        .mat-input-placeholder-wrapper {
+            top: -6px !important;
+        }
+
+        .mat-input-placeholder-wrapper {
```

Comment:
can this style be merged with the same '.mat-input-placeholder-wrapper' one above?